models/uniswap/ethereum/uniswap_ethereum_schema.yml

```diff
@@ -219,4 +219,94 @@ models:
       - *tx_from
       - *tx_to
       - *trace_address
-      - *evt_index
\ No newline at end of file
+      - *evt_index
+
+  - name: uniswap_v3_ethereum_votes
+    meta:
+      blockchain: ethereum
+      sector: dao
+      contributors: soispoke
+    config:
+      tags: ['ethereum', 'votes', 'dao', 'cross-chain']
+    description: >
+      DAO votes on all chains across all contracts and versions
+    columns:
+      - name: blockchain
+      - name: project
+        description: "Project name of the DAO"
+      - name: version
+        description: "Version of the contract built and deployed by the DAO project"
+      - name: block_time
+        description: "UTC event block time of each DAO vote"
+      - name: tx_hash
+        description: "Unique transaction hash value tied to each vote on the DAO"
+      - &dao_name
+        name: dao_name
+        description: "DAO name"
+      - &dao_address
+        name: dao_address
+        description: "DAO wallet address"
+      - &proposal_id
+        name: proposal_id
+      - &votes
+        name: votes
+        description: "Votes weighted by the amount of governance tokens"
+      - &votes_share
+        name: votes_share
+        description: "Votes share in percent for a given proposal"
+      - &token_symbol
+        name: token_symbol
+      - &token_address
+        name: token_address
+      - &votes_value_usd
+        name: votes_value_usd
+        description: "Value in USD corresponding to the governance tokens used to vote on a given proposal"
```

Comment:
is this at the time of the vote or current USD value?

Reply:
At the time of the vote, this is consistent with what we do for nft.trades or dex.trades but I modified the description